### PR TITLE
Start auth documentation, hidden collabo item, conversion script

### PR DESCRIPTION
**Pull request by cskaandorp**
I have written some initial documentation for authentication (README), the collabo menu-item in the profile-popper is finally gone (well, it's hidden), fixed a weird issue with test_asreview_database.py (destroyed the development database for testing purposes) and I have written a small script that may help converting an unauthenticated app into an authenticated one. The simple conversion workflow is described in the README.

I don't know if the README is the proper place for the documentation. Probably not. Feel free to do whatever is necessary to turn the text into proper documentation and let me know if I need to add stuff. 

The conversion script is added in the root folder of the application and is called auth_conversion.py. 

Again, let me know if you need more documentation.

God knows I have pulled flake8 several times over the repo, and isort, and black. And God also knows all my tests are still green.

### CODE REVIEW

File: auth_conversion.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import json
+import sqlite3
+from pathlib import Path
+from uuid import NAMESPACE_URL
+from uuid import uuid5
+
+ASREVIEW_PATH = Path("/Users/casperkaandorp/.asreview")
```
Comment (PeterLombaers) [inline auth_conversion.py:7]: Get the asreview path from asreview:

`[python]
from asreview import asreview_path

ASREVIEW_PATH = asreview_path()
`

File: auth_conversion.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+DATABASE_NAME = "asreview.development.sqlite"
```
Comment (PeterLombaers) [inline auth_conversion.py:8]: Probably when people are actually using this, it should be the production version. Then also the debug parameter in the config should be false.

File: auth_conversion.py (hunk 1 above)
Comment (PeterLombaers) [reply auth_conversion.py:7]: Or better yet, replace the occurences of `ASREVIEW_PATH` by `asreview_path()`

**Comment by PeterLombaers** [comment]
The conversion script works like a charm!

Btw, here is what I used to create the users:

```[python]
import requests

url = 'http://localhost:5000/auth/signup'
data = {
    'password': 'wachtwoord1!',
    'email': 'peter@mail.com',
    'name': 'peter'
}
res = requests.post(url, data=data)
print(res.status_code)
print(res.text)
```
It's probably nice to have a little command line tool to add users. Though that would probably not use the rest api, but just directly access the user database.

**Comment by PeterLombaers** [comment]
If I start the application with `AUTHENTICATION_ENABLE: false` in my config and an empty asreview folder, it works just fine. I can start a project and label some records. However, if I then return to the project overview I get the following:

```
ERROR:asreview.webapp.start_flask:Exception on /api/projects [GET]
Traceback (most recent call last):
  File "C:\Users\Admin\python\asreview\asreview\auth_venv\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Admin\python\asreview\asreview\auth_venv\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Admin\python\asreview\asreview\auth_venv\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\Admin\python\asreview\asreview\auth_venv\lib\site-packages\fl    current_app.login_manager._load_user()
  File "C:\Users\Admin\python\asreview\asreview\auth_venv\lib\site-packages\flask_login\login_manager.py", line 347, in _load_user    raise Exception(
Exception: Missing user_loader or request_loader. Refer to http://flask-login.readthedocs.io/#how-it-works for more info.ERROR:root:Missing user_loader or request_loader. Refer to http://flask-login.readthedocs.io/#how-it-works for more info.
INFO:werkzeug:127.0.0.1 - - [16/Mar/2023 14:45:27] "GET /api/projects HTTP/1.1" 500 -
```

File: auth_conversion.py (hunk 1 above)
Comment (cskaandorp) [reply auth_conversion.py]: I have replaced all occurences of ASREVIEW_PATH with asreview_path

**Comment by J535D165** [comment]
Thanks, merged

**Pull request merged**